freeclimb-cs-sdk/api/call/Call.cs

```diff
@@ -142,6 +145,12 @@ public static Call fromJson(string rawJson)
         /// <returns>The phoneNumberId for this call.</returns>
         public string getPhoneNumberId { get { return this.phoneNumberId; } }
 
+        /// <summary>
+        /// Retrieve  whether the call is active or not from the object.
+        /// </summary>
+        /// <returns>The active property for this call.</returns>
+        public bool getActive { get { return this.active; } }
```

Comment:
"active" is not a call property that is stored anywhere. "active" is true if the call is either ringing, inProgress or queued.